eslint.config.js

```diff
@@ -65,7 +64,6 @@ module.exports = {
         "@typescript-eslint/ban-ts-ignore": "off",
         "@typescript-eslint/no-empty-function": "off",
         "@typescript-eslint/no-explicit-any": "off",
-        "@typescript-eslint/no-floating-promises": 1,
```

Comment:
Not really related to the actual change (and cant comment at right place in code) but do you know what `"@typescript-eslint/no-floating-promises": 2` actually means? [doc](https://typescript-eslint.io/rules/no-floating-promises/) doesn't seem to know numbers as options for this rule.

Reply:
`"@typescript-eslint/no-floating-promises": 1` would warn. I did not find it documented anywhere, but when using a random value, ESLint 8.22.0 prints: 

>Configuration for rule "@typescript-eslint/no-floating-promises" is invalid:
>        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"foo"').

However, it seems like since `"off"` was added below it overrides `1` (`"warn"`). Therefore I removed the line with `1` and kept `"off"`. :slightly_smiling_face:

Reply:
maybe change line 37 to use "error".

Reply:
Good idea. :slightly_smiling_face: While we're at it, I also changed it to `2` for `header/header` and `quotes` to `"error"`. This way it should be easier to read and is in line with the rest of the rules.
Also found where these settings are [documented](https://eslint.org/docs/latest/user-guide/configuring/rules#configuring-rules). 

I also double checked if it still does what it should by modifying some files and running `npm run lint`. :slightly_smiling_face: